registry/remote/repository_test.go

```diff
@@ -2146,3 +2146,63 @@ func Test_ManifestStore_FetchReference(t *testing.T) {
 		t.Errorf("Manifests.FetchReference() = %v, want %v", got, manifest)
 	}
 }
+
+func Test_ZeroSizedBlob_Fetch(t *testing.T) {
```

Comment:
Could we have consistent test names in a pattern of
```
Test_<TypeName>_<FunctionName>_<Scenario>
```
?

For instance, `Test_BlobStore_Fetch_ZeroSizedBlob`?

Reply:
You might also want to put the test asides other `blobStore` tests.

Reply:
Changed the test name to "Test_BlobStore_Fetch_ZeroSizedBlob". Only the blobStore Fetch function has this issue, so only blobStore test is needed. The manifest Fetch function doesn't set Range headers.